dashpay/src/main/kotlin/org/dashj/platform/contracts/wallet/TxMetadata.kt

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Group
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package org.dashj.platform.contracts.wallet
+
+import java.util.Date
+import kotlin.collections.HashMap
+import org.bitcoinj.core.ECKey
+import org.dashj.platform.dapiclient.model.DocumentQuery
+import org.dashj.platform.dpp.document.Document
+import org.dashj.platform.dpp.document.DocumentCreateTransition
+import org.dashj.platform.dpp.document.DocumentsBatchTransition
+import org.dashj.platform.dpp.identifier.Identifier
+import org.dashj.platform.dpp.identity.Identity
+import org.dashj.platform.sdk.platform.Platform
+
+class TxMetadata(
+    val platform: Platform
+) {
```

Comment:
The class is arranged similarly to other Contract related classes where there is a create method, and some get methods.